### DISCUSSION TITLE
e Magick.NET-Q8-AnyCPU nuget package= magick files place in the user \AppData\Local\Temp directory

### DISCUSSION THREAD

**Discussion by stevensrf11** [ImageMagick/ImageMagick#3837] [Development]
I am using the Magick.NET-Q8-AnyCPU nuget package in a Windows C# WPF application .Net Core 3.1
Using this package causes many, and I do say many, magick files being placez in the user \AppData\Local\Temp directory. 
Each file is around 32 Megabytes in size. 
There are four methods called from the the Magick.NET-Q8-AnyCPU:
Read, Composite, Compare, and Signature.

Can anyone tell me why this this Magick Image package create dso many and so huge files in the  \AppData\Local\Temp directory?
Is there a way to prevent this?

**Comment by dlemstra** [comment]
Those files are the native library of ImageMagick that contains all the methods of ImageMagick. That you don't use all of them those not mean they are not included. If you don't want to create those temporary files you probably want to switch to the `Magick.NET-Q8-x64` version instead. And then you will only get the file of the target platform / operating system of your application.
